examples/arkode/CXX_superludist/CMakeLists.txt

```diff
@@ -39,16 +33,6 @@ else()
   include_directories(${MPI_INCLUDE_PATH})
 endif()
```

Comment:
Maybe an item for later, with cmake >= 3.18 I think we can use `MPI::MPI_CXX` (this did not work with CMake < 3.17.3)

Reply:
Yeah, I would like to use it but currently I set it up to only require CMake >= 3.18 if CUDA is on.